### ISSUE TITLE
Wildcard issue

### ISSUE THREAD

**Issue by AresMoused** [ltdrdata/ComfyUI-Impact-Pack#90]
when I attempt to use the wildcard node, some of the __text__ resolve and some didn't. 
I tried :
change name of the wildcard 
add space/delete space before or after the __text__
swap location of the __text__

![image](https://github.com/ltdrdata/ComfyUI-Impact-Pack/assets/108890434/299a0777-358a-444b-8455-a6498bfef564)

**Comment by ltdrdata** [comment]
It's because of new subfolder feature.
wildcard should be now

`__path/wildcardname__`

for example

`__samples/flower__`

I'd updated document, recently.

It seems broken something.
I cannot patch until this weekend.

So try move them to subfolder.

**Comment by AresMoused** [comment]
thanks for the fast response
I tried to put them in a subfolder but it's still not able to pick up the txt file.

But after I tried to place them in the wildcard folder instead of the custom_wildcard folder, its work!!!

tho its weird for the node to pick up some wildcard but not other in the custom_wildcard folde, for example the __eyecolor__ and __haircolor__ contain nearly identical string.

But most importantly its work now. Thanks for the help and the code. Its useful and powerful (the doc are super clear for someone only use Webui before)
